lib/utils.js

```diff
@@ -33,7 +33,8 @@ function debrack (s) {
 }
 
 function uriToFilename (uri, base) {
-  var filename = path.join(base, uri)
+  var decoded = uri.split('/').map(decodeURIComponent).join('/')
```

Comment:
> A goal of this project is to be able to export existing files system users have.

How would this export process work? The existing files would arrive on a Solid server via PUTs and POSTs, right? Which means their filenames would be safely encoded.

> It is NOT a goal to be able to allow clients to put files with characters which are typically illegal in a file system

What would you like to see as the behavior when users/developers inevitably try to PUT or POST those kind of file names (illegal in file system)?